app/mailers/support_mailer.rb

```diff
@@ -5,7 +5,7 @@ def confirm_sign_in(support_user, device:)
 
     notify_email(
       to: support_user.email_address,
-      subject: 'New sign in to Support for Apply for teacher training',
+      subject: 'New sign in for apply for teacher training',
```

Comment:
Is this change right? This notification appears to be related to support. Maybe it should be:

```suggestion
      subject: 'New sign in to Support for Apply',
```

@Tomcowan1

Reply:
Yes that one should change back - thanks, Paul.